src/app/core/entity/schema-datatypes/datatype-number.ts

```diff
@@ -31,6 +31,8 @@ import { EntitySchemaDatatype } from "../schema/entity-schema-datatype";
  */
 export const numberEntitySchemaDatatype: EntitySchemaDatatype = {
   name: "number",
+  viewComponent: "DisplayText",
```

Comment:
Should this not be a separate input type=number?  (e.g. for Educational Materials -> Amount is now accepting strings as well)